content/blog/2017/2017-01-17-Jenkins-is-upgrading-to-Java-8.adoc

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: "Jenkins Upgrades To Java 8"
+tags:
+- java8
+- upgrade
+author: batmat
+---
+
+NOTE: Back in last November, we discussed interesting statistics showing that Jenkins was now running Java 8 on a majority of its running instances.
+This was _52.8%_ back then, and now reaching link:http://stats.jenkins.io/plugin-installation-trend/jvms.json[_58%_ two months later].
+If we only look at Jenkins 2.x, then we reach _72%_.
+
+== Upgrading to Java 8
+
+Balancing those numbers with many other criteria:
+
+* Java 7 has been now end-of-lifed for 18+ months ;
+* People are already moving away from Java 7, as show the numbers ;
+* Java 8 runtime is known from the field to be more stable
+** Many developers have been wanting to be allowed to leverage the improvements that Java 8 provides to the language and platform
+  (lambdas, Date/Time API... just to name a few).
+  Being also a developer community, we want Jenkins to be appealing to contributors.
+
+**So, we have decided that Jenkins core would require Java 8 as a runtime in the next few months.**
+
+See the following for more details about that.
+
+== Timeline
+
+Here is how we plan to roll that baseline upgrade in the next few months.
+
+1. Now: Announce the intention publicly.
+2. April, 2017: Drop support for Jenkins weekly.
+** We will declare what LTS will be the last one to support Java 7.
+3. April-June: First LTS version requiring Java 8 is published.
```

Comment:
May 2017:  Publish first LTS version requiring Java 8.

Pick a month.   You can be wrong, but pick a month.